src/Microsoft.SqlTools.ServiceLayer/Admin/Database/DatabasePrototype.cs

```diff
@@ -892,6 +908,40 @@ public virtual RecoveryModel RecoveryModel
         }
 
         /// <summary>
+        /// The last backup date for the database
+        /// </summary>
+        [Browsable(false)]
+        public DateTime LastBackupDate
+        {
+            get
+            {
+                return this.currentState.lastBackupDate;
+            }
+            set
+            {
+                this.currentState.lastBackupDate = value;
+                this.NotifyObservers();
```

Comment:
I think ideally if this is notifying observers when a property is being updated, it should be implementing the INotifyPropertyChanged interface. But, I also assume this is a huge change, so I guess we can leave it be.